src/init/features/project.ts

```diff
@@ -40,9 +43,9 @@ export async function getProjectInfo(options: any): Promise<ProjectInfo> {
  * @returns A {@link FirebaseProject} object.
```

Comment:
This javadoc `@returns` line looks out-of-date and unhelpful, so let's just remove it.